tensorflow-examples/src/main/java/org/tensorflow/model/examples/mnist/training/LinearRegressionExample.java

```diff
@@ -0,0 +1,154 @@
+/*
+ *  Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *  =======================================================================
+ */
+package org.tensorflow.model.examples.mnist.training;
+
+import org.tensorflow.Graph;
+import org.tensorflow.Session;
+import org.tensorflow.Tensor;
+import org.tensorflow.op.Ops;
+import org.tensorflow.op.core.*;
+import org.tensorflow.op.math.*;
+import org.tensorflow.op.train.ApplyGradientDescent;
+import org.tensorflow.tools.Shape;
+import org.tensorflow.types.TFloat32;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+
+/**
+ * In this example the TensorFlow finds the weight and bias of the linear regression during 1 epoch,
+ * training on observations one by one.
+ *
+ * Also, the weight and bias are extracted and printed.
+ */
+public class LinearRegressionExample {
+    /**
+     * Amount of data points.
+     */
+    private static int N = 10;
```

Comment:
`final` as well? Also, why are we declaring this value as a constant while other constant are mostly just hardcoded below, should we be consistent and declare constants for all of them?

I remember that a guy at Google told me before that their "rule" for this is that if a value is only used once, hardcode it, otherwise create a constant out of it.

Reply:
done